### PR TITLE
Provide instructions to deploy a custom scheduler

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1776803

### CODE REVIEW

File: admin_guide/scheduling/custom_scheduler.adoc [hunk 1]
```diff
@@ -16,14 +16,86 @@ You can run multiple, custom schedulers alongside the default scheduler and conf
...
+    metadata:
+      labels:
+        app: custom-scheduler
+    spec:
+      serviceAccount: custom-scheduler
+      containers:
+        - name: custom-scheduler
+          image: "<namespace>/<image name>:<tag>" <1>
+          imagePullPolicy: Always
+----
```
Comment (geliu2016) [inline admin_guide/scheduling/custom_scheduler.adoc]: LGTM， thanks.

File: admin_guide/scheduling/custom_scheduler.adoc [hunk 2]
```diff
@@ -16,26 +16,99 @@ You can run multiple, custom schedulers alongside the default scheduler and conf
...
+involves creating an image and including that image in a deployment.
+
+. Package your scheduler binary into a container image. 
+
+. xref:../../creating_images/guidelines.html#creating-images-guidelines[Create a container image] containing the scheduler binary. 
++
+For example:
++
+----
+FROM <source>
```
Comment (lamek) [inline admin_guide/scheduling/custom_scheduler.adoc]: Suggestion: update "source" to "Parent image", "base image" or "source image". I checked quickly and I didn't see other examples of how we currently use this, but I was initially confused as to whether this meant source code, source repo, etc.

File: admin_guide/scheduling/custom_scheduler.adoc [hunk 3]
```diff
@@ -16,26 +16,99 @@ You can run multiple, custom schedulers alongside the default scheduler and conf
...
+      serviceAccount: custom-scheduler
+      containers:
+        - name: custom-scheduler
+          image: "<namespace>/<image name>:<tag>" <1>
+          imagePullPolicy: Always
+----
+<1> Specify the container image you created for the custom scheduler.
+
+[[admin-guide-sched-custom-deploy]]
+== Deploying Pods using a Custom Scheduler
```
Comment (lamek) [inline admin_guide/scheduling/custom_scheduler.adoc]: Minor: Pod ---> pod?

It looks like capitalization for the word "pod" is a bit inconsistent throughout.

**Review by lamek — COMMENTED** [review]
2 minor comments and otherwise LGTM.

**Pull request merged**